frontend/src/app/modules/matches/matches-selectors.js

```diff
@@ -21,7 +21,7 @@ const getMatches = state => state.matches.map(match => ({
   team2: fillUsers(match.team2, state),
 }))
 
-const didUserPlayMatch = (userId, match) => {
+export const didUserPlayMatch = (userId, match) => {
```

Comment:
this should ideally return a boolean (`!!allplayers.find()`)

Reply:
done [here](https://github.com/salsita/foosball-rating/pull/92/commits/d7c17da31ec421acbb90f4a19ce22c34412cdf35#diff-08117b849ecc00af7c3c0d609dd86d31)